### ISSUE TITLE
Review Dialect: Gujarati (Python)

### ISSUE THREAD

**Issue by madiedgar** [legesher/legesher-translations#113]
## Review Gujarati Language Translations for the Python Programming Language

Version 1 of Legesher only includes translations for the Python programming language. In order to add this language to the project, complete the steps detailed in [the translation guide](https://legesher.readme.io/docs/translation-guide). 

**Reviewing** a translation is extremely important as we want to best understand the culture and context behind the words we select as keywords for the programming languages. The more perspectives the better! 👀 

>⚠️ Note: this issue is to review the Python keywords and builtin functions only. Any additional translation reviews are encouraged, but not necessary. 

**To complete this issue**:
- [ ] Navigate to the _Python_ section of the dialect's `locale/DIALECT.yml` file
- [ ] Go through each of the translations for the Python keywords and built in functions adding commentary as comments
  - Leave a `✅`after a translation if you agree that's the right word to give context to the keyword
  - Leave a `❓` after a translation if you think there may be a better translation / form of the translation
  - Leave a  `🤔` after a word that you're not sure if there is a translation
- [ ] Create a pull request with changes into branch titled `gujarati-translation` on this repo
- [ ] In your pull request, reference the original pull request that added your dialect. 

### Reviewers Needed
In order to add the translations into our database to use, the translations will need to be approved and accepted by 3 additional native / fluent speakers. If you would like to be one of the reviewers for this language, please mention in this issue! :smile: Also feel free to loop in a friend that might be able to help with the translations!

The translated word needs to have `# ✅✅✅` in the comments after it in order to be accepted. Please take your time in being thorough with these translations. Think back when you were learning how to program and how context of words help you bridge together meaning and functionality. 

**Review Status**:
- [x] `Add New Dialect` Issue (#)   
- [x] Initiator:  (#92)
- [ ] Reviewer 1:
- [ ] Reviewer 2: 
- [ ] Reviewer 3:

**Comment by madiedgar** [comment]
Hey @bvishal8510 👋 , thanks so much for helping with adding the initial Gujarati translation. Do you know anyone else who might be interested in contributing to this?

**Comment by sshiv5768** [comment]
@madiedgar , I am interested.

**Comment by sshiv5768** [comment]
Thanks, @madiedgar for assigning me as a reviewer. 🤗

**Comment by madiedgar** [comment]
Yes please @sshiv5768 👍🏾 we'll need more individuals to add their review, but feel free to add more translations of another programming language.

**Comment by Nidhir2k1** [comment]
Hello @madiedgar I'm interested in contributing.

**Comment by madiedgar** [comment]
@Nidhir2k1 Awesome, I assigned you to the issue. If you need any help, just reach out! 👍

**Comment by 136tejas** [comment]
Hey @madiedgar i would like to contribute to this issue.

**Comment by madiedgar** [comment]
Awesome @136tejas, I'll assign you!

**Comment by 179priyasoni** [comment]
hey @madiedgar I would like to  contribute to this issue.

**Comment by madiedgar** [comment]
Awesome @179priyasoni, I'll assign you to the issue. Let me know if you have any questions. 👍🏾

**Comment by nik132-eng** [comment]
I am new at the Open source but I want to work on this can you please assign me this issue?

**Comment by madiedgar** [comment]
Happy to help walk you through it if you need @nik132-eng 👍

**Comment by nik132-eng** [comment]
yes ma'am I didn't understand how to do it.

**Comment by nik132-eng** [comment]
Please help me @madiedgar

**Comment by madiedgar** [comment]
Join our [discord community](https://discord.gg/DkVjVDP) and I can help walk you through it @nik132-eng 👍

**Comment by aadii0408** [comment]
I would like to contribute to this issues @madiedgar

**Comment by nik132-eng** [comment]
hey @madiedgar is this project participating in hacktoberfest this year?

**Comment by sshiv5768** [comment]
Hey, @Nidhir2k1 yes this project is participating in Hacktoberfest 2023, you can contribute by adding your translations or you can be one of the reviewers(if you are a native Gujarati speaker).
